.github/workflows/go.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16
+        go-version: "1.20"
```

Comment:
I think we should leave this as 1.16, but add >= 1.20 to test that our test is being skipped properly in CI instead of failing. This might require that we rewrite this file slightly to accept a matrix of versions like we have elsewhere.